bower.json

```diff
@@ -1,13 +1,13 @@
 {
-  "name": "adapt-mediaWithBenefits",
-  "version": "1.0.0",
+  "name": "adapt-mediaGallery",
+  "version": "2.0.0",
   "framework": ">=5.3",
-  "homepage": "https://github.com/cgkineo/adapt-mediaWithBenefits",
-  "issues": "https://github.com/cgkineo/adapt-mediaWithBenefits/issues/ ",
-  "component": "mediawithbenefits",
-  "displayName": "Media with benefits",
+  "homepage": "https://github.com/cgkineo/adapt-mediaGallery",
+  "issues": "https://github.com/cgkineo/adapt-mediaGallery/issues/ ",
+  "component": "mediaGallery",
```

Comment:
not 100% sure about this but I _think_ this needs to be lowercase to match the component registration/name